res/css/views/messages/_ReactionsRowButtonTooltip.scss

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2019 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_ReactionsRowButtonTooltip {
+    box-shadow: none;
+    background-color: $reaction-row-button-tooltip-bg-color;
+    color: $reaction-row-button-tooltip-fg-color;
+    text-align: center;
+    font-size: 8px;
+    padding: 6px;
+    border: none;
+    border-radius: 3px;
+
+    .mx_Tooltip_chevron::after {
+        border-right-color: $reaction-row-button-tooltip-bg-color;
+    }
+
+    .mx_ReactionsRowButtonTooltip_reactedWith {
```

Comment:
Does the design imply that this class and mx_ReactionsRowButtonTooltip_senders are inline? It looks a bit weird to me to have a line break in-between.

Reply:
That's how I interpreted it, but I'll ask! 😁

Reply:
Separate line confirmed!